Utils/azure-explorer-common/test/java/com/microsoft/tooling/IntegrationTestBase.java

```diff
@@ -92,23 +88,6 @@ public void setUpStep() throws Exception {
 
         addTextReplacementRule(GLOBAL_ENDPOINT, MOCK_URI + "/");
 
-        int retries = 10;
-        boolean created = false;
-        while (retries > 0) {
-            retries--;
-            try {
-                wireMock = new WireMock(MOCK_HOST, wireMockRule.port());
```

Comment:
wiremock will set localhost as default, if we need to run case on other host, then I should add Mock_Host.